MonoGame.Framework/Input/GamePadThumbSticks.cs

```diff
@@ -63,7 +63,7 @@ internal GamePadThumbSticks(Vector2 leftPosition, Vector2 rightPosition, GamePad
             else
                 ApplySquareClamp();
 
-            SetVirtualButtons(leftPosition, rightPosition);
+            SetVirtualButtons(left, right);

```

Comment:
> It doesn't really matter in practice here

Actually, it does matter. The virtual buttons logic needs the raw values before any deadzone clamping has been applied. The fact that the tests aren't failing just means that I didn't add sufficient coverage for my original fix related to this. Please add the following test cases to the `ThumbStickVirtualButtonsIgnoreDeadZone` test and you should find they fail if the code uses the processed values like it currently is:
```c#
yield return new TestCaseData(
    new Vector2(0.4f, -0.4f), new Vector2(-0.4f, 0.4f), GamePadDeadZone.IndependentAxes,
    Buttons.LeftThumbstickRight | Buttons.LeftThumbstickDown | Buttons.RightThumbstickLeft | Buttons.RightThumbstickUp);

yield return new TestCaseData(
    new Vector2(0.4f, 0f), new Vector2(0f, 0.4f), GamePadDeadZone.Circular,
    Buttons.LeftThumbstickRight | Buttons.RightThumbstickUp);
```